### PR TITLE
Format code using 'cargo fmt'

### PR DESCRIPTION
**Pull request by Atul9**

### CODE REVIEW

File: wgpu-native/src/command/mod.rs [hunk 1]
```diff
@@ -26,18 +26,10 @@ use crate::{
...
 
 use back::Backend;
-use hal::{
-    Device as _,
-    command::RawCommandBuffer,
-};
+use hal::{command::RawCommandBuffer, Device as _};
 use log::trace;
 
-use std::{
```
Comment (kvark) [inline wgpu-native/src/command/mod.rs]: These changes are rather undesired. Is there a way to preserve the old style of imports?

**Comment by kvark** [comment]
Ping! There is one concern, also needs a rebase.

File: wgpu-native/src/command/mod.rs (hunk 1 above)
Comment (Atul9) [reply wgpu-native/src/command/mod.rs]: I will check on this weekend and revert

**Comment by Atul9** [comment]
@kvark I have rebased the branch and updated the PR. I will check if there is a way to preserve the old style of imports.

**Comment by swiftcoder** [comment]
I think you want https://rust-lang.github.io/rustfmt/#imports_layout

**Comment by kvark** [comment]
Thanks @swiftcoder ! "HorizontalVertical" sounds like our choice

**Comment by Atul9** [comment]
@kvark @swiftcoder how about having `imports_layout = "HorizontalVertical"` OR "Vertical" set and running `cargo fmt` on the codebase?

**Comment by kvark** [comment]
Thank you!
This isn't ideal to me, but it looks like it's the best we can do with rustfmt today, so let's proceed.
bors r+

**Pull request merged**